src/libs/ValidationUtils.js

```diff
@@ -230,6 +230,15 @@ function isNumericWithSpecialChars(input) {
     return /^\+?\d*$/.test(input.replace(/[()-]/g, ''));
 }
 
+/**
+ * Checks weather a given first or last name is valid
+ * @param {String} name
+ * @returns {Boolean}
+ */
+function isValidFirstOrLastName(name) {
```

Comment:
also nab I think: would prefer `isValidLengthForFirstOrLastName` as a more descriptive name. I feel like given the current method name I'd assume we'd also check for invalid characters, etc.